src/controller.rs

```diff
@@ -145,3 +181,47 @@ fn apply(config: SsbConfig) -> Option<ApplyResults> {
         None
     }
 }
+
+// Apply the day brightness
+fn apply_day_brightness(config: SsbConfig) -> Option<ApplyResults> {
+    let location = config.location.unwrap_or(Location {
+        latitude: 0.0,
+        longitude: 0.0,
+    });
+    Some(apply_brightness(
+        config.brightness_day,
+        config.brightness_day,
+        0,
+        location,
+        config.overrides,
```

Comment:
realized if  there is an override, it will take that over passing day as night above. So probably need to add an extra argument that forces it to always take day brightness?